datasets/au/oenb_sanctions/au_oenb_sanctions.yml

```diff
@@ -0,0 +1,40 @@
+title: Austrian National Bank Regulations on Terrorism Financing Restrictions
+entry_point: crawler.py
+prefix: au
+coverage:
+  frequency: weekly
+  start: "2024-07-18"
+deploy:
+  schedule: "@weekly"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset compiles Austrian National Bank regulations aimed at combating terrorism through restrictions on capital and payment transactions, 
+  covering the assets of designated individuals under the 2010 Sanctions Act.
+description: |
+  The dataset comprises unofficial versions of regulations provided by the Austrian National Bank (OeNB) aimed at combating terrorism through capital
+  and payment traffic restriction measures. It covers regulations DL 2/2002 and its 8th amendment DL 1/2009. These regulations are considered valid under 
+  the 2010 Sanctions Act and cover assets of specified organizations and individuals.
+publisher:
+  name: Austrian National Bank
+  acronym: OeNB
+  description: |
+    The Oesterreichische Nationalbank (OeNB) is the central bank of the Republic of Austria and as such an integral part of the European System of Central 
+    Banks (ESCB). The OeNB contributes, in the public interest, to monetary and economic policy decision-making in Austria and the euro area. Its core business 
+    areas are cash management, monetary policy, financial stability, statistics, and payments.
+    [Source: Official Website](hhttps://www.oenb.at/en/)
+  url: https://www.oenb.at
+  official: false
```

Comment:
this is official - we've done the data extraction from official documents, right?